modules/tester/manifests/init.pp

```diff
@@ -6,15 +6,23 @@
 		ensure => directory,
 	}
 
+	if versioncmp( "$tester_config[php]", '7.0' ) >= 0 {
+		$phpunit_package_name = "phpunit.phar"
+		$phpunit_repo_url     = "https://phar.phpunit.de/phpunit.phar"
+	} else {
+		$phpunit_package_name = "phpunit-old.phar"
```

Comment:
I did some more testing around this. `5.6+`can use `phpunit.phar` but the older versions need `phpunit-old.phar` so I've adjusted the logic accordingly. Thanks for the review!